### ISSUE TITLE
Update `ws` to next major version (when it's released)

### ISSUE THREAD

**Issue by feross** [feross/bittorrent-tracker#92]
Because of issues like this one: https://github.com/mafintosh/torrent-stream/issues/137#issuecomment-136038039

**Comment by feross** [comment]
Another reason to get rid of this dep is that on an OS X machine with the command line tools installed, but no Xcode, this error is spewed upon installing WebTorrent:

```
npm WARN engine windows-no-runnable@0.0.6: wanted: {"node":"0.6"} (current: {"node":"0.12.7","npm":"2.11.3"})
 
> bufferutil@1.1.0 install /usr/local/lib/node_modules/webtorrent/node_modules/torrent-discovery/node_modules/bittorrent-tracker/node_modules/ws/node_modules/bufferutil
> node-gyp rebuild

xcode-select: error: tool 'xcodebuild' requires Xcode, but active developer directory '/Library/Developer/CommandLineTools' is a command line tools instance

xcode-select: error: tool 'xcodebuild' requires Xcode, but active developer directory '/Library/Developer/CommandLineTools' is a command line tools instance

  CXX(target) Release/obj.target/bufferutil/src/bufferutil.o
  SOLINK_MODULE(target) Release/bufferutil.node

> utf-8-validate@1.1.0 install /usr/local/lib/node_modules/webtorrent/node_modules/torrent-discovery/node_modules/bittorrent-tracker/node_modules/ws/node_modules/utf-8-validate
> node-gyp rebuild

xcode-select: error: tool 'xcodebuild' requires Xcode, but active developer directory '/Library/Developer/CommandLineTools' is a command line tools instance

xcode-select: error: tool 'xcodebuild' requires Xcode, but active developer directory '/Library/Developer/CommandLineTools' is a command line tools instance

  CXX(target) Release/obj.target/validation/src/validation.o
  SOLINK_MODULE(target) Release/validation.node
```

**Comment by No9** [comment]
+1 for removing as many native deps as possible

**Comment by mafintosh** [comment]
We could PR the two native deps to switch to prebuild and then add have them fallback to the js libraries if no prebuild is available

**Comment by feross** [comment]
https://github.com/websockets/ws/issues/577

**Comment by feross** [comment]
The `ws` package just removed its native dependencies, so as soon as they publish that to npm, we can bump the version here and this will no longer be an issue!

https://github.com/websockets/ws/commit/49b11093e9a009e5305dcde7003d3a896b2811dc

**Comment by feross** [comment]
This is done, released as 7.0.0.
